### ISSUE TITLE
Content with charset breaks the responseType

### ISSUE THREAD

**Issue by mtuzinskiy** [cyclosproject/ng-openapi-gen#154]
The `inferResponseType` checks for ending of the media type:
```
if (mediaType.endsWith('/json') || mediaType.endsWith('+json')) {
```
But it's quite usual to have it like this:  
```
  content:
    application/json; charset=utf-8:
      ...
```
Which breaks the inference and makes the `responseType` to be "blob". I guess this can be considered as a bug...
Thanks!

**Comment by mtuzinskiy** [comment]
Changing it to  
```
  if (mediaType.includes('/json') || mediaType.includes('+json')) {
```  
should do the job. I can make a PR if it looks ok

**Comment by luisfpg** [comment]
Yes, please.

**Comment by mtuzinskiy** [comment]
@luisfpg Done! https://github.com/cyclosproject/ng-openapi-gen/pull/155

**Comment by Zryru** [comment]
Was about to open a ticket for this xD

**Comment by luisfpg** [comment]
@Zryru This was already fixed in version 0.17.1.
